documentscanner/src/main/res/values/strings_document_scanner.xml

```diff
@@ -0,0 +1,26 @@
+<resources>
+    <string name="app_name">MEGA Document Scanner</string>
+
+    <string name="scan_title_crop" context="Screen title for cropping current scan">Crop</string>
+    <string name="scan_title_save_scan" context="Screen title for saving current scan">Save scan</string>
+    <string name="scan_action_add" context="Action text to add a new page">Add</string>
+    <string name="scan_action_save" context="Action text to save document">Save</string>
+    <string name="scan_action_retake" context="Action text to retake current scan picture">Retake</string>
+    <string name="scan_file_name" context="Document scanner file name">File name</string>
+    <string name="scan_file_type" context="Input field description in the document save screen">File type</string>
+    <string name="scan_quality" context="Input field description in the document save screen">Quality</string>
+    <string name="scan_destination" context="Input field description in the document save screen">Destination</string>
+    <string name="scan_file_type_pdf" context="Document scanner PDF file type option">PDF</string>
+    <string name="scan_file_type_jpg" context="Document scanner JPG file type option">JPG</string>
```

Comment:
These two strings are no translatables, so they should not be included in the strings file that we will upload to Babel. They should be converted as constants or moved to a separated file (for example `strings_donottranslate.xml`) like in the MEGA Android app project.